index.js

```diff
@@ -49,6 +49,11 @@ function wpOAuth(client_id, opts){
     response_type: opts.response_type || 'token'
   };
 
+	// include state if in opts
```

Comment:
The block is misaligned. AFAIK we are using spaces instead of tabs. Let's keep in this way until we change the whole style code.